src/components/CallView/LocalMediaControls.vue

```diff
@@ -212,6 +238,14 @@ export default {
 			return (this.model.attributes.localScreen || this.splitScreenSharingMenu) ? t('spreed', 'Screensharing options') : t('spreed', 'Enable screensharing')
 		},
 
+		screenSharingButtonAriaLabel() {
+			if (this.screenSharingMenuOpen) {
+				return ''
+			}
+
+			return (this.model.attributes.localScreen || this.splitScreenSharingMenu) ? t('spreed', 'Screensharing options') : t('spreed', 'Enable screensharing')
```

Comment:
```suggestion
			return (this.model.attributes.localScreen || this.splitScreenSharingMenu)
				? t('spreed', 'Screensharing options')
				: t('spreed', 'Enable screensharing')
```
Cleaner to me, but personal preferences, I don't know how you do it in talk :man_shrugging: